samples/error-correction/syndrome/syndrome.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""
+Script for running Pseudo-syndrome example as defined in Syndrome.qs.
+This script takes the number of data qubits as input.
+
+Example usage:
+    python syndrome.py -q 5
+"""
+
+import numpy as np
+import qsharp
+import argparse
+
+from Microsoft.Quantum.Samples.Hardware.Syndrome import SamplePseudoSyndrome
+
+parser = argparse.ArgumentParser(
+    prog="PseudoSyndrome",
+    description="Program for running a PseudoSyndrome circuit that detects random errors on qubits in gate sequence."
+)
+parser.add_argument("-q", "--qubits", type=int, default=1)
+
+if __name__ == "__main__":
+    args = parser.parse_args()
+    paulis = ["PauliX", "PauliY", "PauliZ"]
```

Comment:
Could you use `[qsharp.Pauli.X, qsharp.Pauli.Y, qsharp.Pauli.Z]` here?

Reply:
Yes! Thanks!

Reply:
If you want to get really cute, you could do something like `paulis = list(map(qsharp.Pauli, range(1, 4)))`.

(I don't really recommend that.)

Reply:
It looks like the above suggestions don't work because of a bug in `iqsharp`. I've filed a ticket here: https://github.com/microsoft/iqsharp/issues/256

Reply:
Good catch @guenp, thanks for finding and filing that!